docs/concepts/policy/pod-security-policy.md

```diff
@@ -420,16 +420,24 @@ root filesystem (i.e. no writeable layer).
 
 ### FlexVolume drivers
 
-When the [`Volumes`](#volumes-and-file-systems) field contains `flexVolume` in
-its list value, the cluster admin can further specify which driver(s) is permitted
-by setting the `allowedFlexVolumes` field.
+This specifies a whiltelist of flex volume drivers that are allowed to be used
+by flexVolume. An empty list or nil means there is no restriction on the drivers.
+Please make sure [`volumes`](#volumes-and-file-systems) field  contains the
+`flexVolume` volume type, no FlexVolume driver is allowed otherwise.
 
-**AllowedFlexVolumes** - Provides a whitelist of allowed FlexVolumes. Empty or
-nil indicates that all FlexVolume drivers may be used. For example, the following
-setting only permits the `examle/fast_cache` driver to be used on nodes:
+For example:
 
 ```yaml
-allowedFlexVolumes: [ "example/fast_cache" ]
+apiVersion: extensions/v1beta1
```

Comment:
This looks correct, but it's not a valid PSP object since PSP requires a few other fields:

```yaml
apiVersion: extensions/v1beta1
kind: PodSecurityPolicy
metadata:
  name: allow-flex-volumes
spec:
  fsGroup:
    rule: RunAsAny
  runAsUser:
    rule: RunAsAny
  seLinux:
    rule: RunAsAny
  supplementalGroups:
    rule: RunAsAny
  volumes:
    - flexVolume
  allowedFlexVolumes: 
    - driver: example/lvm
    - driver: example/cifs
```

Maybe just note that it's missing some other required fields?

```yaml
apiVersion: extensions/v1beta1
kind: PodSecurityPolicy
metadata:
  name: allow-flex-volumes
spec:
  # ... other spec fields
  volumes:
    - flexVolume
  allowedFlexVolumes: 
    - driver: example/lvm
    - driver: example/cifs
```

Reply:
Thanks for the comments. Will change based on your second proposal. After all, this section is more focused on the `allowedFlexVolumes` and related properties.